Package.swift

```diff
@@ -1,19 +1,23 @@
-// swift-tools-version:5.6
+// swift-tools-version:5.8
 
 import PackageDescription
 
 let package = Package(
     name: "FuturedKit",
     platforms: [
-        .iOS(.v13),
+        .iOS(.v16),
         .macOS(.v10_15),
-        .watchOS(.v6),
-        .tvOS(.v13)
+        .watchOS(.v9),
+        .tvOS(.v16)
     ],
     products: [
         .library(
-            name: "FuturedKit",
-            targets: ["FuturedKit"]
+            name: "FuturedKitArchitecture",
```

Comment:
Nepojmenujeme to spis `FuturedArchitectureKit` a treba `FuturedUtilityKit`/`FuturedHelperKit`? Prijde mi to tak trosku prirozenejsi a ze to ma vic stavy. 😎

Reply:
souhlas 👍

Reply:
nebo jeste jako napad: dat z nazvu knihoven pryc to "Kit". Ten Package zustane FuturedKit ale knihovny se budou jmenovat `FuturedArchitecture` a `FuturedHelpers`